src/drivers/pfsoc_crypto/crypto.c

```diff
@@ -0,0 +1,202 @@
+/****************************************************************************
+ *
+ *   Copyright (c) 2020 Technology Innovation Institute. All rights reserved.
```

Comment:
2022?

Reply:
Oops. well, merged already. Let's fix in some following PR